conf/incremental.json

```diff
@@ -21,7 +23,8 @@
     },
     "incremental": {
       "mode": "incremental"
-    }
+    },
+    "privatization": "none"
```

Comment:
Is there a specific reason to turn off default privatization for incremental? It was on before.

Reply:
The incremental mode has not been tested with the mutex analysis yet (also not before) which would be required for privatization. I think there might be a similar location dependence as in the mallocWrapperAnalysis which would need to be removed to make it incrementally runable. Testing the incremental mode with more analyses should definitely be done. I turned off the privatization for now, because it can not be turned on anymore without the mutex analysis activated.